modules/rubiconBidAdapter.js

```diff
@@ -411,6 +415,47 @@ RubiconAdapter.masSizeOrdering = function(sizes) {
     });
 };
 
+/**
+ * syncEmily
+ * @summary A user sync dependency for the Rubicon Project adapter
+ * When enabled, creates an user sync iframe after a delay once the first auction is complete.
+ * Only fires once except that with each winning creative there will be additional, similar calls to the same service.
+ * @example
+ *  // Config example for Rubicon user sync
+ *  $$PREBID_GLOBAL$$.rubiconGlobals = {
```

Comment:
@mkendall07 happy to do that, but when do you expect the `setConfig` api to be merged?  We were hoping to get this merged very soon.